CurrentTime.js

```diff
@@ -0,0 +1,32 @@
+function showTime(){
+    let date = new Date();
+    let tm = [
+        date.getHours(), 
+        date.getMinutes(), 
+        date.getSeconds(),
+        date.toLocaleString('default', {weekday: 'long'}), 
+        date.getDate(),
+        date.toLocaleString('default', {month: 'long' })];
+
+    for (let i = 0; i < 3; i++) {
+        if (tm[i] < 10) {
+            tm[i] = "0" + tm[i].toString();
+        }
+        else {
+            tm[i] = "" + tm[i].toString();
+        }
+    }
+    
+    let time = tm[0] + ":" + tm[1];
+    let days = tm[3] + ", " + tm[4] + " " + tm[5];
```

Comment:
This solution is okey, but a little old-style. In ES6 we have string literals which is supported in majority of browsers. Also you should use `const` because you never mutate this variables after initialization.

```suggestion
    const time = `${tm[0]}:${tm[1]}`;
    const days = `${tm[3]}, ${tm[4]} ${tm[5]}`;
```